src/core/internal/backtrace/elf.d

```diff
@@ -31,6 +31,11 @@ else version (OpenBSD)
     import core.sys.openbsd.sys.elf;
     version = LinuxOrBSD;
 }
+else version (Solaris)
+{
+    import core.sys.solaris.sys.elf;
```

Comment:
Some enum values in the elf_i386 and elf_sparc modules need moving to the common elf module for this to compile.

Reply:
`SHF_ORDERED` and `SHF_EXCLUDE` are the definitions to move out of elf_386 and elf_SPARC to the elf module.

Reply:
Missing definition of `SHF_COMPRESSED` in the elf module too.